README

```diff
@@ -280,16 +280,16 @@ SYSV INIT.D SCRIPTS:
         needs to look like, and provide an implementation at the marked places.
 
 WARNINGS:
-        systemd will warn you during boot if /usr is on a different
-        file system than /. While in systemd itself very little will
-        break if /usr is on a separate partition, many of its
-        dependencies very likely will break sooner or later in one
-        form or another. For example, udev rules tend to refer to
-        binaries in /usr, binaries that link to libraries in /usr or
-        binaries that refer to data files in /usr. Since these
-        breakages are not always directly visible, systemd will warn
-        about this, since this kind of file system setup is not really
-        supported anymore by the basic set of Linux OS components.
+        systemd will warn during boot if /usr is not pre-mounted already during
+        early boot (that means: either located on the same file system as / or
+        mounted in the initrd already). While in systemd itself very little
```

Comment:
"or already mounted in initramfs"? ("already" at the end of a sentence rubs me the wrong way.)